Clarify: Use a proper web server in prod, and don't interpret var (#530)
ezsystems/developer-documentation: docs/getting_started/install_ez_platform.md

    For security reasons, in production web server should not have write access to other directories than `var`. Skip the step above and follow the link below for production needs instead.

>     You must also make sure that the web server cannot interpret files in the `var` directory through PHP. Follow the instructions on [setting up a virtual host](#set-up-virtual-host), as these take care of this for you.

Reviewer: Not sure if we need the second sentence, "Set up virtual host" is four lines down, and it's part of the procedure anyway, so it's not likely to be missed.

Author: If we just remove the 2nd sentence, people will wonder "how do I make sure of this?". We got a bug report due to someone not following those virtual host steps, that's why I wanted to clarify it. There's an important security aspect to this. It can be rewritten, of course.

>     You must also make sure that the web server cannot interpret files in the `var` directory through PHP. Follow the instructions on [setting up a virtual host](#set-up-virtual-host), as these take care of this for you.

Reviewer: ```suggestion
    You must also make sure that the web server cannot interpret files in the `var` directory through PHP. To do so, follow the instructions on [setting up a virtual host below](#set-up-virtual-host).
```
